src/lib/configuration.rs

```diff
@@ -0,0 +1,61 @@
+use std::env;
+
+#[derive(Debug)]
+pub struct Configuration {
+    pub nun_user: String,
+    pub nun_pwd: String,
+    pub nun_dbs_dir: String,
+    pub nun_ws_addr: String,
+    pub nun_http_addr: String,
+    pub nun_tcp_addr: String,
+    pub nun_replicate_addr: String,
+    pub nun_log_level: String,
+    pub nun_log_dir: String,
+    pub nun_log_to_file: String,
+}
+
+#[cfg(debug_assertions)]
+fn expect_env_var(name: &str, default: &str) -> String {
+    return env::var(name).unwrap_or(String::from(default));
+}
+
+#[cfg(not(debug_assertions))]
+fn expect_env_var(name: &str, _default: &str) -> String {
+    return env::var(name).expect(&format!(
```

Comment:
This is causing problems in release builds
e.g.: 
```
./target/release/nun-db start -u mateus -p mateus
thread 'main' panicked at 'Environment variable NUN_USER is not defined: NotPresent', src/lib/configuration.rs:24:27
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

I did not provide the env var in this example, but I passed the parameter to the command line. So it should work as expected since the user and PWD are provided via the command line. 

Env vars are optional if (and only if) the USER and PWD are provided via command line

Reply:
noted will make commandline as priority #1